docs/docs/guides/create-personal-access-token.mdx

```diff
@@ -3,22 +3,23 @@ id: create-personal-access-token
 title: Create And Use Personal Access Tokens
 ---
 
-Head over to the "Personal Access Token" page
+Head over to the "Personal Access Tokens" page and click on the plus symbol to
+create a new token.
 
-![Personal Access Token list](../images/get-started/pat-list.png)
```

Comment:
Is the removal here intentional?

Reply:
Yes, I was switchting it out for pat-create.png which is a bit more zoomed in on the relevant part.